sys/vtimer/vtimer.c

```diff
@@ -32,9 +34,16 @@
 
 #include "vtimer.h"
 
-#define ENABLE_DEBUG    (0)
```

Comment:
Ah now I see why you did that. Would you mind moving the include also upwards? (Sorry, I know this is a bit picky :-) )